Ginger/GingerCore/Drivers/WebServicesDriverLib/HttpWebClientUtils.cs

```diff
@@ -84,13 +84,25 @@ public bool RequestContstructor(ActWebAPIBase act, string ProxySettings,bool use
 
             //Headers
             AddHeadersToClient();
-            
+
+            //SetAutoDecomression
+            SetAutoDecomression();
+                        
             if (act.GetType() == typeof(ActWebAPISoap))
                 return RequestConstracotSOAP((ActWebAPISoap)act);
             else
                 return RequestConstractorREST((ActWebAPIRest)act);
         }
 
+        private void SetAutoDecomression()
```

Comment:
Please correct the typo in Decompression

Reply:
above changes pushed